Resources/Prototypes/CosmaticDrift/Entities/Structures/Machines/cryo_sleep_pod.yml

```diff
@@ -0,0 +1,31 @@
+﻿- type: entity
+  id: MachineCryoSleepPod
+  parent: BaseMachine
+  name: cryosleep chamber
+  description: Perfect for having a 10,000 year nap.
+  components:
+    - type: Sprite
+      sprite: Structures/cryosleep_pod.rsi
+      drawdepth: Mobs
+      layers:
+      - state: sleeper_1
+    - type: InteractionOutline
+    - type: Anchorable
+      delay: 999999
```

Comment:
I think people should be able to move these, if anyone hides them or something an admin can make some more